src/vmm/src/vstate/memory.rs

```diff
@@ -242,10 +241,12 @@ impl GuestMemoryExtension for GuestMemoryMmap {
                     .regions
                     .iter()
                     .map(|r| {
-                        f.try_clone().map(|file_clone| {
-                            let offset = FileOffset::new(file_clone, r.offset);
+                        let fo = f.try_clone().map(|file_clone| {
```

Comment:
nit: something a bit more informational than `fo`?

Reply:
mh, this is just a transient state, as this function is removed in a later commit, so I think this is fine